Microsoft.Recognizers.Resources/Microsoft.Recognizers.Resources.csproj

```diff
@@ -102,6 +102,12 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>xcopy "$(TargetDir)$(TargetFileName)" "$(TargetDir)..\..\..\$(SolutionName)\$(OutDir)" /Y
+				xcopy "$(TargetDir)$(TargetName).pdb" "$(TargetDir)..\..\..\$(SolutionName)\$(OutDir)" /Y

```

Comment:
Only need to copy dlls. Just as this PR https://github.com/Microsoft/Recognizers-Text/pull/48/files